signal/signal.go

```diff
@@ -42,7 +42,8 @@ func systemdNotifyReady() error {
 	if notified {
 		log.Info("Systemd was notified about our readiness")
 	} else {
-		log.Info("We're not running within systemd")
+		log.Info("We're not running within systemd or the service"+
```

Comment:
nit: add space after word "service". Also this needs to be run through the formatter, the linter is complaining.

Reply:
That's weird, it seems to require a space in front of `+` but there are other lines without space and no complain. :man_shrugging:

Reply:
Yes, that's a bit weird. I think if there are no additional arguments on the follow-up line (only the string is being continued), then it wants a space. If there are more arguments, no space is required.